### PR TITLE
Book List and Nav Bar

### PR DESCRIPTION
**Pull request by Gianni-Nicolas**
**Summary**

Create Book list and nav bar in Index.html

**Screenshot**

Captura de Pantalla 2020-02-16 a la(s) 17 06 16

**Trello Card**

https://trello.com/c/4Ko58GJH/30-listado-de-libros

https://trello.com/c/zLhM6Waq/31-navbar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,99 @@
   <title>HTML & CSS Training</title>
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <link rel="stylesheet" type="text/css" href="css/application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1>
+  <nav class="navbar">
+    <a class="wolox-image" href="#" target="_self"> </a>
```
Comment (martinmona) [inline index.html]: I think it'd be better like an `img` tag, and then, if you need it to behave as a link then wrap it inside an` a` tag

File: scss/book-list.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+}
+
+.book-list-container {
+  align-items: center;
+  background-color: $light-grey;
+  display: flex;
+  flex-wrap: wrap;
+  height: 100vh;
+  justify-content: center;
+  overflow-y: scroll;
```
Comment (martinmona) [inline scss/book-list.scss]: don't use it here

File: scss/book-list.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  background-size: contain;
+  height: 70px;
+  max-height: 100%;
+  width: 215px;
+}
+
+.logout {
+  color: $black;
+  font-size: 18px;
+  height: 18px;
```
Comment (martinmona) [inline scss/book-list.scss]: not necessary

File: scss/book-list.scss [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  line-height: 21px;
+  width: 63px;
```
Comment (martinmona) [inline scss/book-list.scss]: not necessary

File: scss/book-list.scss [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+.book-detail-link {
+  background-color: white;
+  box-shadow: 4px 4px 10px 0 rgba(0,0,0,0.5);
+  height: 300px;
+  margin: 20px 70px;
+  width: 198px;
+}
+
+.book-list-img {
+  max-width: 100%;
```
Comment (martinmona) [inline scss/book-list.scss]:
```suggestion
  width: 100%;
```

**Review by martinmona — CHANGES_REQUESTED** [review]

**Review by martinmona — APPROVED** [review]
:clap:

File: scss/book-list.scss [hunk 6]
```diff
@@ -0,0 +1,79 @@
+@import "colors";
+
+.navbar {
+  align-items: center;
+  background-color: white;
```
Comment (nvescio) [inline scss/book-list.scss:5]: Use variable here

**Review by nvescio — CHANGES_REQUESTED** [review]

**Review by nvescio — APPROVED** [review]

**Pull request merged**